spec/features/user_signs_in_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature 'user signs in', %Q{
+  As a user
+  I want to sign in
+  So that I can track my park activities
+} do
+
+  scenario 'an existing user specifies a valid email and password' do
+    user = FactoryGirl.create(:user)
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Sign In'
+    expect(page).to have_content('Welcome Back!')
+    expect(page).to have_content('Sign Out')
+  end
+  scenario 'a nonexistant email and password is supplied' do
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: 'nobody@example.com'
+    fill_in 'Password', with: 'password'
+    click_button 'Sign In'
+
+    expect(page).to have_content('Invalid Email or password.')
+    expect(page).to_not have_content('Welcome Back!')
+    expect(page).to_not have_content('Sign Out')
+  end
+  scenario 'a exisiting email with the wrong password is denied access' do
+    user = FactoryGirl.create(:user)
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: 'incorrectPass'
+    click_button 'Sign In'
+    expect(page).to have_content('Invalid Email or password.')
+    expect(page).to_not have_content('Sign Out')
+  end
+  scenario 'an already authenticated user cannont re-sign in' do
+    user = FactoryGirl.create(:user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Sign In'
+
```

Comment:
Totes agree with Nick here! You could also extract this granular sign in instructions into a `before(:each)` block!